src/BloomExe/Publish/MakePdfUsingGeckofxHtmlToPdfProgram.cs

```diff
@@ -38,7 +38,8 @@ class MakePdfUsingGeckofxHtmlToPdfProgram
 			var runner = new CommandLineRunner();
 			string exePath;
 			var bldr = new StringBuilder();
-			var loc = Assembly.GetExecutingAssembly().Location;
+			// Codebase is reliable even when Resharper copies the EXE somewhere else for testing.
+			var loc = Assembly.GetExecutingAssembly().CodeBase.Substring("file:///".Length);
```

Comment:
This change is really nothing to do with this commit except I needed it to get unit tests passing locally with Resharper